modules/virt-networking-wizard-fields-web.adoc

```diff
@@ -22,10 +22,7 @@
 |List of available network attachment definitions.
 
 |Type
-|List of available binding methods. For the default pod network, `masquerade`
-is the only recommended binding method. For secondary networks, use the `bridge`
-binding method. The `masquerade` method is not supported for non-default
-networks.
+|List of available binding methods. For the default pod network, `masquerade` is the only recommended binding method. For secondary networks, use the `bridge` binding method. The `masquerade` method is not supported for non-default networks. If you configured an SR-IOV network device and defined an SR-IOV network in the namespace, select `SR-IOV` to attach a virtual function to the virtual machine.
```

Comment:
> select `SR-IOV` to attach a virtual function to the virtual machine.

what's a `virtual function`?

Reply:
@aburdenthehand Can you please comment on "virtual function". Or can we simplify and state:
select SR-IOV for the virtual machine.

Reply:
IMHO omitting it is best, because it can be "to attach a virtual network device" instead of "to attach a virtual function" but we already know it's a "[virtual] network device", so I think it's safe to just omit.

Reply:
@yaacov - that sounds good. I will redo the sentence and omit that later part.